integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -315,6 +315,26 @@ void testAlterTableUpdateColumnType() {
     checkTableColumns(tableName, updateColumns, getTableInfo(tableName));
   }
 
+  @Test
+  void testAlterTableRenameColumn() {
+    String tableName = "test_rename_column";
+    dropTableIfExists(tableName);
+    List<SparkColumnInfo> simpleTableColumns = getSimpleTableColumn();
+    createSimpleTable(tableName);
+    checkTableColumns(tableName, simpleTableColumns, getTableInfo(tableName));
+
+    String oldColumnName = "age";
+    String newColumnName = "new_age";
+
+    sql(
+        String.format(
+            "ALTER TABLE %S RENAME COLUMN %S TO %S", tableName, oldColumnName, newColumnName));
+    ArrayList<SparkColumnInfo> renameColumns = new ArrayList<>(simpleTableColumns);
+    renameColumns.remove(SparkColumnInfo.of(oldColumnName, DataTypes.IntegerType, null));
```

Comment:
here is a little tricky,  the test could pass only if "age" is the last column. may be you could add a column first and then rename it for simple, like #2502